gum_test.html

```diff
@@ -212,12 +224,18 @@ <h4>getUserMedia is missing!</h4>
     try {
       const stream = await navigator.mediaDevices[gum](constraints);
       message.innerHTML = "<p>Success!</p>";
-      if (stream.getVideoTracks().length) {
+      const [audioTrack] = stream.getAudioTracks();
+      const [videoTrack] = stream.getVideoTracks();
+      if (audioTrack) {
+        controls.appendChild(elementMuted.parentNode);
+        audio.muted = elementMuted.checked;
+        video.muted = elementMuted.checked;
+      }
+      if (videoTrack) {
         content.appendChild(video);
-        if (stream.getAudioTracks().length) {
-          controls.appendChild(mute);
-          video.muted = muted.checked;
-        }
+        controls.appendChild(camOn.parentNode);
+        camOn.onclick = () => { videoTrack.enabled = camOn.checked; };
+        camOn.onclick();
```

Comment:
can't the checked state change through other means than click? keyboard navigation and selecting it with space is what I'm thinking. The "change" event looks like the right thing. Also applies to `micOn` and `elementMuted`.